rpc/lib/server/handlers.go

```diff
@@ -445,9 +445,6 @@ func (wsc *wsConnection) OnStart() error {
 
 // OnStop unsubscribes from all events.
```

Comment:
Is this comment still correct? We no longer unsubscribe from within this function. Should we?